pelican/utils.py

```diff
@@ -330,20 +330,17 @@ def walk_error(err):
                 dst_path = os.path.join(dst_dir, o)
                 if os.path.isfile(src_path):
                     logger.info('Copying %s to %s', src_path, dst_path)
-                    copy_file_metadata(src_path, dst_path)
+                    copy_file(src_path, dst_path)
                 else:
                     logger.warning('Skipped copy %s (not a file or '
                                    'directory) to %s',
                                    src_path, dst_path)
 
 
-def copy_file_metadata(source, destination):
-    '''Copy a file and its metadata (perm bits, access times, ...)'''
-
-    # This function is a workaround for Android python copystat
-    # bug ([issue28141]) https://bugs.python.org/issue28141
+def copy_file(source, destination):
+    '''Copy a file'''
     try:
-        shutil.copy2(source, destination)
+        shutil.copyfile(source, destination)
```

Comment:
I think the point of using `shutil.copy2` was to actually preserve metadata, specifically creation/modification times, to assist in HTML/Browser caching. Otherwise static files will be re-dated with each site generation and browsers will consider that these files are new and re-request them even if they were not changed. Permission transfer is an unfortunate side effect of `shutil.copystat`, but we can pick and choose what to copy if we do it ourselves. So this needs to be followed by a [`os.utime`](https://docs.python.org/3/library/os.html#os.utime) call to preserve that behavior.